src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/http_client.html.twig

```diff
@@ -54,6 +54,18 @@
                         {% else %}
                             <h4>Requests</h4>
                             {% for trace in client.traces %}
+                                {% set profiler_token = '' %}
+                                {% set profiler_link = '' %}
+                                {% if trace.info.response_headers is defined %}
+                                    {% for header in trace.info.response_headers %}
+                                        {% if header matches '/x-debug-token: .*$/i' %}
```

Comment:
the regexp should be anchored: `'/^x-debug-token: .*$/i'` - same below

Reply:
I agree, I have anchored the regexes.